src/fireEvent.ts

```diff
@@ -1,31 +1,18 @@
 import { ReactTestInstance } from 'react-test-renderer';
-import { TextInput } from 'react-native';
 import act from './act';
 import { isHostElement } from './helpers/component-tree';
-import { filterNodeByType } from './helpers/filterNodeByType';
 import { getHostComponentNames } from './helpers/host-component-names';
 
 type EventHandler = (...args: any) => unknown;
 
-const isTextInput = (element?: ReactTestInstance) => {
-  if (!element) {
-    return false;
-  }
-
-  // We have to test if the element type is either the `TextInput` component
-  // (for composite component) or the string "TextInput" (for host component)
-  // All queries return host components but since fireEvent bubbles up
-  // it would trigger the parent prop without the composite component check.
-  return (
-    filterNodeByType(element, TextInput) ||
```

Comment:
Please correct me if I'm wrong but we can remove this check since all queries return host components(AFAIR that has been added recently)

Reply:
Yes, the composite `TextInput` check is being removed, and we only leave the host `"TextInput"` check.